logging/monolog_console.rst

```diff
@@ -122,5 +122,148 @@ Now, log messages will be shown on the console based on the log levels and verbo
+                    # verbosity_levels:
+                    #     VERBOSITY_NORMAL: NOTICE
+
+    .. code-block:: xml
+
+        <!-- config/packages/test/monolog.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:monolog="http://symfony.com/schema/dic/monolog"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <monolog:config>
+                <!-- ... -->
+
+                <monolog:handler name="console" type="console" process-psr-3-messages="false">
+                    <monolog:channels>
+                        <monolog:channel>!event</monolog:channel>
+                        <monolog:channel>!doctrine</monolog:channel>
+                        <monolog:channel>!console</monolog:channel>
+                    </monolog:channels>
+                </monolog:handler>
+                
+                <monolog:handler name="test" type="test" level="debug" />
+            </monolog:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/test/monolog.php
+        $container->loadFromExtension('monolog', array(
+            'handlers' => array(
+                'console' => array(
+                   'type' => 'console',
+                   'process_psr_3_messages' => false,
+                   'channels' => array('!event', '!doctrine', '!console'),
+                ),
+                'test' => array(
+                   'type' => 'test',
+                   'level' => 'debug'
+                ),
+            ),
+        ));
+
+Finally, access the :class:`Monolog\\Handler\\Testhandler` and validate the log output::
+
+    // tests/Command/CreateUserCommandTest.php
+    namespace App\Tests\Command;
+
+    use App\Command\CreateUserCommand;
+    use Symfony\Bundle\FrameworkBundle\Console\Application;
+    use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+    use Symfony\Component\Console\Tester\CommandTester;
+    use Monolog\Handler\TestHandler;
+
+    class CreateUserCommandTest extends KernelTestCase
+    {
+        public function testExecute()
+        {
+            $kernel = static::createKernel();
+            $application = new Application($kernel);
+
+            $command = $application->find('app:create-user');
+            $commandTester = new CommandTester($command);
+            $commandTester->execute(array(
+                'command'  => $command->getName(),
+
+                // pass arguments to the helper
+                'username' => 'Wouter',
+
+                // prefix the key with two dashes when passing options,
+                // e.g: '--some-option' => 'option_value',
+            ));
+            
+            $logger = $command->getLogger();
```

Comment:
From here it could be:
```php
$testLogs = self::$container->get(TestHandler::class);

$this->assertTrue($logs->hasRecordThatContains('Some info', Logger::DEBUG), 'ASSERT_ERROR_MESSAGE_1');
$this->assertTrue($logs->hasRecordThatContains('Some more info', Logger::NOTICE), 'ASSERT_ERROR_MESSAGE_2');
$this->assertFalse($logs->hasRecordThatContains('Command Success', Logger::INFO), 'ASSERT_ERROR_MESSAGE_3');
```
We should then target 4.4.